src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ChangeDirectionDueDateConfirmation.java

```diff
@@ -27,20 +28,37 @@ public PostSubmitCallbackResponse handle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+        final String hoChangeDirectionDueDateInstructStatus =
+            asylumCase.read(AsylumCaseFieldDefinition.HOME_OFFICE_CHANGE_DIRECTION_DUE_DATE_INSTRUCT_STATUS,
+                String.class).orElse("");
+
         PostSubmitCallbackResponse postSubmitResponse =
             new PostSubmitCallbackResponse();
 
-        String directionsTabUrl =
-            "/case/IA/Asylum/"
-            + callback.getCaseDetails().getId()
-            + "#directions";
-
-        postSubmitResponse.setConfirmationHeader("# You have changed the due date");
-        postSubmitResponse.setConfirmationBody(
-            "#### What happens next\n\n"
-            + "The party has been notified of their time extension. You can see the status of the direction in the "
-            + "[directions tab](" + directionsTabUrl + ")"
-        );
+        if (hoChangeDirectionDueDateInstructStatus.equalsIgnoreCase("FAIL")) {
+
+            postSubmitResponse.setConfirmationBody(
+                "![Respondent notification failed confirmation]"
+                + "(https://raw.githubusercontent.com/hmcts/ia-appeal-frontend/master/app/assets/images/respondent_notification_failed.svg)\n"
+                + "#### Do this next\n\n"
+                + "Contact the respondent to tell them what has changed, including any action they need to take.\n"
+            );
+
+        } else {
```

Comment:
This would be an issue if the first request fails. All of the susequent requests would show this message.

Reply:
Added new status